### ISSUE TITLE
px-backup fails for non ssl backup

### ISSUE THREAD

**Issue by whoatemyjam** [portworx/helm#357]
**Is this a BUG REPORT or FEATURE REQUEST?**:
BUG REPORT
**What happened**:
Added  s3 backup storage without ssl and by checking disable ssl button and it got validated  but backups are failing.

" Error uploading resources: blob (key "backup/bkup-6-435607e/6ad9dfc6-88db-4a3f-bb23-72984c9737a7/namespaces.json") (code=Unknown): RequestError: send request failed caused by: Put "https://bastion.dev1.ocp4.baremetal.xyz:8020/portworx/dbackup/bkup-6-435607e/6ad9dfc6-88db-4a3f-bb23-72984c9737a7/namespaces.json": http: server gave HTTP response to HTTPS client "
**What you expected to happen**:
with  ssl is disabled  in backup location ,  the backup  must complete instead of trying to go via https
**How to reproduce it (as minimally and precisely as possible)**:
create a s3 bucket , in our case scality cloudserver, go to add cluster in px-backup, add backup location and credentials in cloud setting and perform backup of any object 
**Anything else we need to know?**:
is  there a way  to add self signed certs in px-backup  config  so backups can be performed using them
**Environment**:
- Container Orchestrator and version: openshift 4.9
                                                            oc version
                                                            Client Version: 4.9.13
                                                            Server Version: 4.9.13
                                                            Kubernetes Version: v1.22.3+e790d7f

- Cloud provider or hardware configuration: Azure
- OS (e.g. from /etc/os-release):  Coreos
- Kernel (e.g. `uname -a`):
- Install tools: helm 
- Others: px-backip app version 2.2.0,

**Comment by diptiranjanpx** [comment]
Yes, there has been a regression in 2.2.0 version which is going to be fixed as part of 2.2.1 release.
